### DISCUSSION TITLE
type numbers in api as numbers and not as strings

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#2726] [Help]
how to make numeric fields return to api as numbers and not as strings?

**Comment by thetutlage** [comment]
Can you share more information please? I am not sure which fields are you talking about

**Reply by sasha172017** [reply]
before last updates i received data from model (v5.0.4)

``created_at: "28.02.2021 08:18"``
``deleted_at: null``
``id: 42``
``image_url: "'``
``name: "Name1"``
``position: 4``
``slug: "name1"``
``updated_at: "14.05.2021 12:10"``

Now l receive data from model

``created_at: "28.02.2021 08:18"``
``deleted_at: null``
``id: "42"``
``image_url: "'``
``name: "Name1"``
``position: "4"``
``slug: "name1"``
``updated_at: "14.05.2021 12:10"``

type number (id, position) i receive how string from model but i need to get these how number

**Reply by mastermunj** [reply]
Which database are you using?

**Reply by sasha172017** [reply]
i found it) it is because i use bigint. If use integer in database (pgsql) i receive in number. But i need bigint and i need to receive in number or bigint data, is it impossible?

**Reply by mastermunj** [reply]
For postgres, add these lines in your `server.ts` and `ace` files, just after all imports / require statements end.

```ts
const pg = require('pg');
pg.types.setTypeParser(20, parseInt);
```
